Formula/go.rb

```diff
@@ -19,7 +17,7 @@ class Go < Formula
       sha256 "6ebbafcac53bbbf8c4105fa84b63cca3d6ce04370f5a04ac2ac065782397fc26"
     end
 
-    go_version = "1.6"
+    go_version = "1.7"
```

Comment:
Please also update the `:revision` should be updated to `26c35b4dcf6dfcb924e26828ed9f4d028c5ce05a` 🎱